rpmlb/cli.py

```diff
@@ -73,6 +73,22 @@
     '--copr-repo', '-C', metavar='COPR_REPO',
     help='Target Copr for copr builder.',
 )
+@click.option(
+    '--epel', metavar='VERSION', type=int, default=7,
```

Comment:
> Sorry my suggesting name `--koji-epel` was not good.

As per above comment – lets leave it at `koji-epel` for now, and replace it when the platform specifiers are implemented.